modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DestinationType.java

```diff
@@ -1,10 +1,24 @@
 package org.wso2.andes.kernel;
 
+import org.wso2.andes.amqp.AMQPUtils;
+import org.wso2.andes.kernel.router.AndesMessageRouter;
+
 /**
  * Defines different destination types for each protocol.
  */
 public enum DestinationType {
-    TOPIC,
-    QUEUE,
-    DURABLE_TOPIC
+
+    TOPIC(AMQPUtils.TOPIC_EXCHANGE_NAME),
+    QUEUE(AMQPUtils.DIRECT_EXCHANGE_NAME),
+    DURABLE_TOPIC(AMQPUtils.TOPIC_EXCHANGE_NAME);
+
```

Comment:
These enums were defined exactly because we wanted to remove AMQP dependency from andes core. Looks it was re-introduced. :(